### ISSUE TITLE
Wrapping quotations

### ISSUE THREAD

**Issue by matthiasbeyer** [mgeisler/textwrap#121]
Hi. Thanks for developing this, it is really useful!

I have an issue: When wrapping the following content:

```
> foo bar baz
> baz baz
```

for example, I end up with

```
> foo
bar
baz
> baz
baz
```

But I want the quotes to be fixed, of course:

```
> foo
> bar
> baz
> baz
> baz
```

Could you please add support for this? It is useful for wrapping Email quotations for example, but I think it could also be useful for wrapping markdown and other formats.

**Comment by mgeisler** [comment]
Hey Matthias,

Handling this kind of prefixes would indeed be cool! I would probably suggest taking a look at how Emacs handles this: the word wrapping in Emacs is quite clever and will adaptively handle things like (hanging) indentation, comments, and bullet lists.

**Comment by mgeisler** [comment]
Fixed with the release of 0.13.3 and the new `refill` function. This function can rewrap text like the above (but only one paragraph at a time). Improvement PRs would be very welcome to make it even more flexible.
